kafka-connect/resources/kubernetes.yaml

```diff
@@ -7,16 +7,16 @@ metadata:
     type: deployment
     kind: kafka-connect
 data:
-  nodes: "3"
+  nodes: "1"
```

Comment:
I don't know too much about Kafka Connect but, is it normal having 1 node but then something line `KAFKA_CONNECT_CONFIG_STORAGE_REPLICATION_FACTOR` to 3 ?

Reply:
The `KAFKA_CONNECT_CONFIG_STORAGE_REPLICATION_FACTOR` relates to Kafka nodes. Not to KAfka Connect nodes. So it is fine to have single Kafka Connect node and yet use `3` in these settings. Current master has also only single Kafka Connect node as default.

Reply:
My ignorance ;)